posthog/queries/funnels/test/__snapshots__/test_funnel.ambr

```diff
@@ -116,6 +116,16 @@
 # name: TestFOSSFunnel.test_funnel_with_precalculated_cohort_step_filter.2
   '
   
+  SELECT count(1)
+  FROM cohortpeople
+  WHERE team_id = 2
+    AND cohort_id = 2
+    AND version < NULL
```

Comment:
this looks like a bug, we probably shouldn't check for stale cohorts when version is <= 0 or None